src/librustdoc/html/render/mod.rs

```diff
@@ -921,7 +923,7 @@ fn render_assoc_item(
         w.reserve(header_len + "<a href=\"\" class=\"fnname\">{".len() + "</a>".len());
         write!(
             w,
-            "{indent}{vis}{constness}{asyncness}{unsafety}{defaultness}{abi}fn <a href=\"{href}\" class=\"fnname\">{name}</a>\
+            "{indent}{vis}{constness}{asyncness}{unsafety}{defaultness}{abi}fn <a {href} class=\"fnname\">{name}</a>\
```

Comment:
Are you sure this is correct? I don't think it's valid to have a link without an href.

Reply:
It's not the prettiest, but according to https://www.w3schools.com/tags/att_a_href.asp it's valid.

Reply:
Ok. Can you add that link as a comment?

Reply:
added 👍